release/release_data_tests.yaml

```diff
@@ -60,7 +61,9 @@
 - name: write_parquet
   run:
     timeout: 3600
-    script: python read_and_consume_benchmark.py s3://ray-benchmark-data/parquet/1TiB --format parquet --write
+    script: > 
+      python read_and_consume_benchmark.py 
+      s3://ray-benchmark-data/tpch/parquet/sf1000/lineitem --format parquet --write
```

Comment:
Drive-by change to update the read, write, and count datasets to be more canonical (e.g., TPCH rather than my random synthetic dataset)